dpp/src/main/proto/wallet-utils.proto

```
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package org.dashj.platform.wallet;
```

Buf (1.57.2)

Files with package "org.dashj.platform.wallet" must be within a directory "org/dashj/platform/wallet" relative to root but were in directory "dpp/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)